bin/propolis-server/src/lib/lib.rs

```diff
@@ -1,10 +1,21 @@
+// always present
 pub mod config;
+pub mod mock_server;
```

Comment:
Should this also be behind the `mock-only` flag?

Reply:
i still wanted it being pulled in for unit tests, though i suppose i can do `#[cfg(any(test, feature = "mock-only"))]` instead